content/features/featuresDeepDive/materials/using/masterPBR.md

```diff
@@ -96,6 +96,49 @@ Once the conversion done, let's see the custom options available on this version
+
+Babylon.js materials constructed as metallic-roughness can also exert control over specular highlights through several parameters on `PBRMaterial`. Beyond the basic setup of the material, these parameters need to be defined to shape specular reflections.
+
+- **metallicF0Factor**: This is the same as `specularFactor` from `KHR_materials_specular`. This parameter is multiplied with `metallicReflectanceColor` to control the intensity of specular reflections.
+- **metallicReflectanceColor**: This is the same as `specularColorFactor` from `KHR_materials_specular`. This parameter sets specular color at F0. As surface normals move toward F90, this value is interpolated to white.
+- **metallicReflectanceTexture**: This is a texture that holds values for `metallicReflectanceColor` in the RGB channels and values for `metallicF0Factor` in the Alpha channel. The texture components are multiplied with `metallicReflectanceColor` and `metallicF0Factor` when calculating the final specular contribution. This texture can be used as a single input for both `metallicReflectanceColor` and `metallicF0Factor` if authoring a `PBRMaterial` right in engine. However, if working with textures from a glTF file authored with the `KHR_material_specular` extension, this parameter acts as `specularTexture` as defined in the extension. This is because the extension defines `specularTexture` and `specularColorTexture` as two different textures with `specularTexture` holding the strength of specular reflections in the alpha channel. If working with `KHR_material_specular` textures, assign the texture defined as `specularTexture` to the `metallicReflectanceTexture` parameter and set the `useOnlyMetallicFromMetallicReflectanceTexture` parameter to true.
+- **useOnlyMetallicFromMetallicReflectanceTexture**: If this parameter is set to true, only the Alpha channel of `metallicReflectanceTexture` is used in lighting calculations. This parameter is important if using textures authored for the `KHR_material_specular` extension because if both `metallicReflectanceTexture` and `reflectanceTexture` are assigned textures, `metallicReflectanceTexture` will be used by default unless `useOnlyMetallicFromMetallicReflectanceTexture` is set to true. If the parameter is set to true, the RGB channels of the texture assigned to `reflectanceTexture` will be used to determine values for `metallicReflectanceColor`.

```

Comment:
I'd put this last, as it refers to `reflectanceTexture`, which hasn't yet been explained.

Reply:
done